ToolBox/SECDataDownloader/SECDataDownloader.cs

```diff
@@ -89,6 +124,17 @@ public void Download(string rawDestination, DateTime start, DateTime end)
                             Log.Trace($"SECDataDownloader.Download(): Downloading temp filing archive to: {tmpFile}");
                             client.DownloadFile($"{BaseUrl}/Feed/{currentDate.Year}/{quarter}/{currentDate:yyyyMMdd}.nc.tar.gz", tmpFile);
 
+                            var tmpFileStat = new FileInfo(tmpFile);
+                            var tmpFileSizeInKB = tmpFileStat.Length / 1024;
+
+                            // Have max and low be +-1% of the stated file size
+                            if (tmpFileSizeInKB > fileSizeInKB + (fileSizeInKB * 0.01m) || tmpFileSizeInKB < fileSizeInKB - (fileSizeInKB * 0.01m))
```

Comment:
why the 1% buffer?

Reply:
Stated file size might be a little skewed. I've observed a few files being 1-5kb larger or smaller from the file explorer. I figured 1% is small enough to be a reasonable lower and upper bound to allow room for flexibility and not have it fail if it's not exactly the same amount of KB as SEC says they are